sr/tools/cli/inv_mv.py

```diff
@@ -2,13 +2,16 @@
 
 
 def command(args):
+    import os
+    import os.path
```

Comment:
Do we need `os.path` imported?

Reply:
Ah, good spot. We don't. Fixed in 2655309.